### PR TITLE
Add support for tkn clustertask logs

### LINKED ISSUE #1070: ClusterTask logs support in tkn
### Feature request

In order to maintain the consistency, we should add `tkn ct logs` which will interactively ask for the clustertask name and then interactively ask for the taskrun associated with that task or `tkn ct logs <ctname>` which will ask for taskrun associated with it.

### Use case

Being able to see the logs corresponding to a particular clustertask just like we get the logs for `task` or `pipeline`

### UI Example

```
$ tkn clustertask logs check-make 
? Select taskrun: check-make-run-w5lrt started 1 minute ago
[lint-makefile] checkmake.
[lint-makefile] 
[lint-makefile]   Usage:
[lint-makefile]   checkmake [options] <makefile>
[lint-makefile]   checkmake -h | --help
[lint-makefile]   checkmake --version
[lint-makefile]   checkmake --list-rules
[lint-makefile] 
[lint-makefile]   Options:
[lint-makefile]   -h --help               Show this screen.
[lint-makefile]   --version               Show version.
[lint-makefile]   --debug                 Enable debug mode
[lint-makefile]   --config=<configPath>   Configuration file to read
[lint-makefile]   --format=<format>       Output format as a Golang text/template template
[lint-makefile]   --list-rules            List registered rules
```
OR
```
$ tkn clustertask logs
? Select clustertask: check-make
? Select taskrun: check-make-run-w5lrt started 2 minutes ago
[lint-makefile] checkmake.
[lint-makefile] 
[lint-makefile]   Usage:
[lint-makefile]   checkmake [options] <makefile>
[lint-makefile]   checkmake -h | --help
[lint-makefile]   checkmake --version
[lint-makefile]   checkmake --list-rules
[lint-makefile] 
[lint-makefile]   Options:
[lint-makefile]   -h --help               Show this screen.
[lint-makefile]   --version               Show version.
[lint-makefile]   --debug                 Enable debug mode
[lint-makefile]   --config=<configPath>   Configuration file to read
[lint-makefile]   --format=<format>       Output format as a Golang text/template template
[lint-makefile]   --list-rules            List registered rules
```

### PR DESCRIPTION
**Pull request by vinamra28**
# Changes

In order to maintain the consistency in commands adding `tkn clustertask logs`
subcommand which will display the logs of the taskrun corresponding to the
particular clustertask.
- `tkn clustertask logs` will interactively ask for the clustertask and
  then the corresponding taskrun
- `tkn clustertask logs name` will interactively ask to select the
  taskrun whose logs we want to see in default namespace
- `tkn clustertask logs name -n namespace` will show the logs of the
  taskrun present in particular namespace.
    
Signed-off-by: vinamra28 <vinjain@redhat.com>

Closes #1070 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```release-note
Add logs subcommand which will display logs of the selected clustertask and corresponding taskrun
```

### CODE REVIEW

File: pkg/cmd/clustertask/logs_test.go [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+			wantError: true,
+			want:      "Error: clustertasks.tekton.dev \"notexist\" not found\n",
+		},
+		{
+			name:      "Specify notexist taskrun name",
+			command:   []string{"logs", "task", "notexist", "-n", "ns"},
+			input:     cs,
+			dc:        dc1,
+			wantError: true,
+			want:      "Error: Unable to get Taskrun: taskruns.tekton.dev \"notexist\" not found\n",
```
Comment (mpetason) [inline pkg/cmd/clustertask/logs_test.go:265]:
```suggestion
			want:      "Error: Unable to get TaskRun: taskruns.tekton.dev \"notexist\" not found\n",
```

File: pkg/cmd/clustertask/logs_test.go [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
```
Comment (mpetason) [inline pkg/cmd/clustertask/logs_test.go:97]:
```suggestion
			want:      "Error: Unable to get TaskRun: taskruns.tekton.dev \"notexist\" not found\n",
```

File: pkg/cmd/clustertask/logs_test.go [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+		Follow: false,
+		Limit:  5,
+	}
+	err = getAllInputs(lopt)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+
+	if lopt.ClusterTaskName != taskName {
+		t.Error("No auto selection of the first clustertask when we have only one")
```
Comment (mpetason) [inline pkg/cmd/clustertask/logs_test.go:205]:
```suggestion
		t.Error("No auto selection of the first ClusterTask when we have only one")
```

**Review by mpetason — COMMENTED** [review]
Small updates for consistency. Not sure if it is required - just noticed different case usage.

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

**Pull request merged**